bin/tag-generator.py

```diff
@@ -4,7 +4,7 @@
 ALPINE_VERSIONS = ['3.16', '3.17', '3.18', '3.19']
 
 LTS_VERSIONS = [ "8", "11", "17", "21"]
-AL2023_GENERIC_VERSIONS = ["20"]
+AL2023_GENERIC_VERSIONS = ["22"]
```

Comment:
Original intention of this was to have a AL2023 image with the generic linux package, which we don't need anymore since we have the native packages.